src/combinatorics/simple_graph/order.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright (c) 2021 Eric. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+import combinatorics.simple_graph.subgraph
+
+/-!
+# Order relation on simple graphs
+
+This file gives `simple_graph` a `bounded_lattice` instance under the `is_subgraph` relation, and
+proves a couple lemmata about how this relation interacts with `simple_graph.subgraph.spanning_coe`.
+
+# Main definitions:
+
+* `simple_graph.is_subgraph`: A `simple_graph` is a subgraph of another if its `adj` is dominated by
+  the other graph's.
+
+* `bounded_lattice` instance: With this definition, `simple_graph` forms a `bounded_lattice`.
+
+* `simple_graph.to_subgraph`: If a `simple_graph` is a subgraph of another, then you can turn it
+  into a member of the larger graph's `simple_graph.subgraph` type.
+
+-/
+
+namespace simple_graph
+
+variable {V : Type*}
+
+/-- The relation that one `simple_graph` is a subgraph of another. -/
+def is_subgraph (x y : simple_graph V) := ∀ {v w : V}, x.adj v w → y.adj v w
+
+/-- The union of two `simple_graph`s. -/
+def union (x y : simple_graph V) : simple_graph V :=
+{ adj := λ v w, x.adj v w ∨ y.adj v w,
+  sym := λ v w h, by rwa [x.adj_comm, y.adj_comm] }
+
+/-- The intersection of two `simple_graph`s. -/
+def inter (x y : simple_graph V) : simple_graph V :=
+{ adj := λ v w, x.adj v w ∧ y.adj v w }
+
+instance : has_union (simple_graph V) := ⟨union⟩
+instance : has_inter (simple_graph V) := ⟨inter⟩
```

Comment:
IMO, having these instances is just confusing - better to just stick to sup / inf than have two ways to spell the same thing.

Reply:
I did this because that's what `subgraph` does; @kmill @hmonroe thoughts?

Reply:
Oh, I consider that to be a mistake in `subgraph`.  Would you mind renaming them in both places?

Reply:
sure, I'll remove them then :)